src/toolchain_cmd.go

```diff
@@ -414,7 +414,7 @@ func installGoToolchain() error {
 		return skippedToolchain{toolchain, "no GOPATH set (assuming Go is not installed and you don't want the Go toolchain)"}
 	}
 
-	srclibpathDir := filepath.Join(strings.Split(srclib.Path, ":")[0], toolchain) // toolchain dir under SRCLIBPATH
+	srclibpathDir := filepath.Join(filepath.SplitList(srclib.Path)[0], toolchain) // toolchain dir under SRCLIBPATH
```

Comment:
Do these also need a `if srclib.Path == "" { return errors.New("srclib.Path is empty") }` guard?